ldms/man/ldmsd_yaml_parser.rst

```diff
@@ -23,27 +23,30 @@ SYNOPSIS
 DESCRIPTION
 ===========
 
-A single ldmsd can be configured using a YAML configuration file. This
-can be done by running ldmsd directly, or generating a configuration
-parser with the ldmsd_yaml_parser
+A single ldmsd daemon configuration script or a fleet of them can be generated from
+an ldms YAML configuration file. When ldmsd is configured with a -y option, the
+ldmsd daemon uses ldmsd_yaml_parser to translate the yaml input.
 
 LDMSD YAML OPTIONS
 ==================
 
-**-y,**\ *CONFIG_PATH*
-   The path to the YAML configuration file.
+**-y,--ldms_config**\ *CONFIG_PATH*
+   The path to the YAML configuration file. This is required.
 
-**-n,**\ *NAME*
-   The name of the LDMS daemon.
+**-n,--daemon_name**\ *NAME*
+   The name of the single LDMS daemon to generate.
 
-LDMSD_YAML_PARSER COMMAND SYNTAX
-================================
+**-p,--generate_config_path**\ *PATH*
+   The name of an existing directory to dump fleet of files.
 
-**--ldms-config,**\ *CONFIG_PATH*
-   The path to the YAML configuration file.
+**-d,--debug**
+   Increase the level of debug output to stdout.
```

Comment:
Debug messages, warnings, errors, etc. should all go to stderr, not stdout.

Reply:
if @nick-enoent is ok with adding logger use, i can update all the debug print related stuff to logger.